pkg/controller/controller_instance.go

```diff
@@ -2020,3 +2023,16 @@ func shouldStartOrphanMitigation(statusCode int) bool {
 		statusCode == http.StatusRequestTimeout ||
 		is5XX
 }
+
+// removeServiceInstanceFailedCondition removes the Failed condition from the
```

Comment:
So, when an instance which provisioned but then failed an update is then updated again, we remove the failed condition?  maybe we just shouldn't set it when updates fail.  WDYT?